File: 2278-percentage-of-letter-in-string/2278-percentage-of-letter-in-string.swift
class Solution {
    func percentageLetter(_ s: String, _ letter: Character) -> Int {
        var counterLetter = 0
        s.forEach({it in
            if it == letter {
                counterLetter += 1
            }
        })
        
        if counterLetter == 0 {
            return 0
        }
        
        let result = Double(counterLetter) / Double(s.count) * 100.0
        return Int(floor(result))
    }
}

File: 2278-percentage-of-letter-in-string/2278-percentage-of-letter-in-string.rs
impl Solution {
    pub fn percentage_letter(s: String, letter: char) -> i32 {
        let mut counter = 0;

        for c in s.chars().into_iter() {
            if c == letter {
                counter += 1;
            }
        }

        (counter as f64 / s.len() as f64 * 100_f64).floor() as i32
    }
}
